crates/compiler/load_internal/src/file.rs

```diff
@@ -4127,234 +4163,7 @@ fn build_header<'a>(
             };
 
             let module_id = module_ids.get_or_insert(&pq_module_name);
-            imported_modules.insert(module_id, region);
```

Comment:
Most of this logic was duplicated with `build_header`, so it has been consolidated there.